From 7a36897350bcdd549058db70a9e15c9c8d5a4456 Mon Sep 17 00:00:00 2001
From: Kai Luo <lkail@cn.ibm.com>
Date: Mon, 10 Jul 2023 16:01:27 +0800
Subject: [PATCH 75/87] UPSTREAM: [JITLink][PowerPC] Fixed unused variable
 warning. NFC.

(cherry picked from commit 79786c4d23f1fd7af438e4fd4e33ec109626bee4)
---
 llvm/include/llvm/ExecutionEngine/JITLink/ppc64.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/llvm/include/llvm/ExecutionEngine/JITLink/ppc64.h b/llvm/include/llvm/ExecutionEngine/JITLink/ppc64.h
index 333adbe6a9b0..d4b7256e563a 100644
--- a/llvm/include/llvm/ExecutionEngine/JITLink/ppc64.h
+++ b/llvm/include/llvm/ExecutionEngine/JITLink/ppc64.h
@@ -254,6 +254,7 @@ inline Error applyFixup(LinkGraph &G, Block &B, const Edge &E,
       uint32_t NopInst = support::endian::read32<Endianness>(FixupPtr + 4);
       assert(NopInst == 0x60000000 &&
              "NOP should be placed here for restoring r2");
+      (void)NopInst;
       // Restore r2 by instruction 0xe8410018 which is `ld r2, 24(r1)`.
       support::endian::write32<Endianness>(FixupPtr + 4, 0xe8410018);
     }
